src/content/docs/setup/configuration/aem-prerender.mdx

```diff
@@ -0,0 +1,552 @@
+Modern storefronts use client-side JavaScript to render product information, limiting discoverability by search engines and AI systems that don't execute JavaScript. Prerendering solves this by generating complete HTML with structured data before pages are requested.
+
+**Key Benefits:**
+
+- **Enhanced SEO**: Complete HTML for search engines improves indexing and rankings
+- **LLM Readability**: Structured markup that AI systems can parse and understand
+- **Improved Performance**: Reduces the time-to-first-byte and serves immediate content
+- **Natural Fallback**: Pre-rendered content displays while client-side drop-ins load
+- **Automated Updates**: Monitors the product catalog and updates markup when products change
+
+## Implementation roadmap
+
+This guide walks you through implementing the prerender solution in both required repositories. Follow these steps in order:
+
+<Diagram 
+  type="mermaid" 
+  caption="Complete implementation workflow showing the step-by-step process for setting up prerendered product pages in your storefront."
+  code={`%%{init: {'theme':'base', 'themeVariables': {'fontSize':'16px'}}}%%
+flowchart LR
+    Start([<strong>Start Implementation</strong>]):::start
+    
+    Step1["<strong>1. Check Prerequisites</strong><br/>Verify access, tools, and storefront requirements"]:::step
+    Step2["<strong>2. Understand the Architecture</strong><br/>Learn how the two repositories work together"]:::step
+    Step3["<strong>3. Install & Deploy App Builder App</strong><br/>Repository: aem-commerce-prerender<br/>Clone, configure, and deploy to App Builder"]:::repo1
+    Step4["<strong>4. Integrate with Your Storefront</strong><br/>Repository: Your Commerce Boilerplate<br/>Modify product-details block to use prerendered markup"]:::repo2
+    Step5["<strong>5. Customize (Optional)</strong><br/>Adjust templates, schemas, and queries"]:::optional
+    
+    End([<strong>✓ Implementation Complete</strong>]):::complete
+    
+    Start --> Step1
+    Step1 --> Step2
+    Step2 --> Step3
+    Step3 --> Step4
+    Step4 --> Step5
+    Step5 --> End
+    
+    classDef start fill:#e3f2fd,stroke:#1976d2,stroke-width:3px,color:#000
+    classDef step fill:#f5f5f5,stroke:#666,stroke-width:2px,color:#000
+    classDef repo1 fill:#fff3e0,stroke:#f57c00,stroke-width:3px,color:#000
+    classDef repo2 fill:#e8f5e9,stroke:#388e3c,stroke-width:3px,color:#000
+    classDef optional fill:#fce4ec,stroke:#c2185b,stroke-width:2px,color:#000,stroke-dasharray: 5 5
+    classDef complete fill:#c8e6c9,stroke:#2e7d32,stroke-width:3px,color:#000
+    
+    linkStyle default stroke-width:3px,stroke:#666
+`}
+/>
+
+## Prerequisites
+
+Before implementing the prerender solution, verify you have:
+
+### Required Access and Permissions
+
+- **Adobe Developer Console access** with "Developer" role for your organization
+- **App Builder workspace** (Stage and Production workspaces recommended)
+- **AEM Admin API access** for your Edge Delivery Services project
```

Comment:
Consider including a reference to the AEM Admin API documentation: https://www.aem.live/docs/admin.html#tag/status/operation/status